### PR TITLE
Feature/nox

### LINKED ISSUE #42: Add PMO
Add capability to compute climate impacts of Primary Mode Ozone (PMO)

### PR DESCRIPTION
**Pull request by stefan-voelk**
## Before you get started
- [x] 🙋‍♀️ Create an issue to discuss what you are going to do!

---

## Description
- Closes #42 

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. 

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation change

## How has this been tested?
Please describe the software tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] calc_ch4_test.py: TestCalcPmoRF with testing valid input and missing CH4 input

**Test configuration**:
* Operating system: Windows 11 Enterprise (64-Bit)
* Other relevant information:

## Checklist
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any changed dependencies have been added or removed correctly

### CODE REVIEW

File: tests/calc_ch4_test.py [hunk 1]
```diff
@@ -31,3 +32,40 @@ def test_empty_conc_dict(self):
             oac.calc_ch4_rf(
                 config, conc_dict, conc_ch4_bg_dict, conc_n2o_bg_dict
             )
+
+
+@pytest.fixture(name="create_rf_dict", scope="class")
+def fixture_load_inv():
+    """Create exampel dictionary with computed RF values
```
Comment (liammegill) [inline tests/calc_ch4_test.py:39]: Spelling mistake: "example"

File: openairclim/calc_response.py [hunk 2]
```diff
@@ -131,3 +141,31 @@ def calc_resp_all(config, resp_dict, inv_dict):
             out_inv_dict[year] = out_arr
         out_dict[spec] = out_inv_dict
     return out_dict
+
+
+def calc_resp_sub(config, species_sub):
+    """
+    Calculates responses for specified sub-species.
```
Comment (liammegill) [inline openairclim/calc_response.py:148]: I would add some more description explaining that the results for the main species must be available and calculated before the results for the sub-species can be calculated (that's what I understand this function requires).

**Review by liammegill — CHANGES_REQUESTED** [review]
Two very small comments. Otherwise looks good and I'm happy to approve the merge!

File: openairclim/calc_response.py (hunk 2 above)
Comment (stefan-voelk) [reply openairclim/calc_response.py:148]: This make sense. description added.

File: tests/calc_ch4_test.py (hunk 1 above)
Comment (stefan-voelk) [reply tests/calc_ch4_test.py]: changed in current version

**Review by liammegill — APPROVED** [review]
Looks good to me!

**Pull request merged**